code/modules/mob/living/simple_animal/friendly/cockroach.dm

```diff
@@ -19,6 +19,8 @@
 	ventcrawler = VENTCRAWLER_ALWAYS
 	gold_core_spawnable = FRIENDLY_SPAWN
 	var/squish_chance = 50
+	var/isdancer = FALSE
```

Comment:
```suggestion
	var/isdancer = TRUE
```
Just for the sake of the event. Otherwise, since the chance is so, low it'd never see the light of day!